libs/parent-add-activity/feature/src/lib/parent-add-activity.component.html

```diff
@@ -0,0 +1,32 @@
+<ion-content>
+    <ion-header [translucent]="true">
+        <ion-toolbar>
+          <ion-title class="ion-text-center"> <h1 class="text-white ion-text-center">The Au Pair</h1> </ion-title>
+        </ion-toolbar>
+      </ion-header>
+    <h2>Add Activity</h2>
+    <div class="activity-details">
+        <form class = "form-details">
+
+            <!-- <ion-item >
+                <ion-label  position="floating">Activity Name</ion-label>
+                <ion-input class = "input"></ion-input>
+            </ion-item> -->
+
+            <input type="text" placeholder="Activity Name">
+            <input type="text" placeholder="Description"> 
+            <input type="text" placeholder="Location"> 
+            <input type="text" placeholder="Day Of The Week"> 
+            <input type="text" placeholder="Time Slot"> 
+            <input type="text" placeholder="budget"> 
+            <input type="text" placeholder="Child"> <br/>
+            <input class = "addActivity" type="submit" value="Add Activity">
+        </form>
+    </div>
+    <img id="active-2" src="assets/images/active.png">
```

Comment:
We need to maybe in a future branch find a way to place these on all screens so we don't have to add them manually every time.